src/main/java/com/project/videotecha/config/SecurityConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.project.videotecha.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class SecurityConfiguration {
```

Comment:
Just a recommendation to maybe rename this class to WebSecurityConfig to align it with the standard spring documentation naming convention